docs/data-sources/helm_feature.md

```diff
@@ -0,0 +1,73 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "tanzu-mission-control_helm_feature Data Source - terraform-provider-tanzu-mission-control"
+subcategory: ""
+description: |-
+  
+---
+
+# tanzu-mission-control_helm_feature (Data Source)
+
```

Comment:
Please add content from official docs. 

You can add, this data source allows us to enable or disable the feature and ensure proper cleanup on disable.

Reply:
It's a data source and we only read with the help of data source.

Will take up helm documentation gap in the next PR